src/components/templates/pipelines.ts

```diff
@@ -0,0 +1,76 @@
+export const newPipelineTemplate = `
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: new-pipeline
+spec:
+  params:
+    - name: paramName
+      type: string
+  resources:
+    - name: app-git
+      type: git
+    - name: app-image
+      type: image
+  tasks:
+    - name: build-app
+      taskRef:
+        name: s2i-java-11
+        kind: ClusterTask
```

Comment:
We should update this Pipeline template, as it uses ClusterTask, which is already deprecated and removed.